### PR TITLE
Adding images [WAIV-1047]

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/object/AppendForm.js [hunk 1]
```diff
@@ -648,6 +652,7 @@ export default class AppendForm extends Component {
   };
 
   getFieldRules = fieldName => {
+    // debugger;
```
Comment (milklim) [inline src/client/object/AppendForm.js]: remove debug

File: src/client/styles/custom.less [hunk 2]
```diff
@@ -10,6 +10,7 @@
 @black: #000;
 @black-text: rgba(0, 0, 0, 0.84);
 @black-text-basic: rgba(0, 0, 0, 0.65);
+//@gray-text-basic: rgba(0, 0, 0, 0.45);
```
Comment (milklim) [inline src/client/styles/custom.less]: need to remove?

File: src/client/object/AppendForm.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/object/AppendForm.js]: ++

File: src/client/styles/custom.less (hunk 2 above)
Comment (konstantin1390) [reply src/client/styles/custom.less]: ++

**Pull request merged**